packages/foreman/python-keycloak-httpd-client-install/python-keycloak-httpd-client-install.spec

```diff
@@ -0,0 +1,50 @@
+# Created by pyp2rpm-3.3.2
+%global pypi_name keycloak-httpd-client-install
+
+Name:           python-%{pypi_name}
+Version:        1.2
+Release:        1%{?dist}
+Summary:        Tools to configure Apache HTTPD as Keycloak client
+
+License:        GPLv3
+URL:            https://github.com/latchset/keycloak-httpd-client-install
+Source0:        https://files.pythonhosted.org/packages/source/k/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+ 
+BuildRequires:  python2-devel
+BuildRequires:  python2-setuptools
+
+%description
+
+
+%package -n     python2-%{pypi_name}
```

Comment:
Hm. [`setup.py` defines requires](https://github.com/latchset/keycloak-httpd-client-install/blob/master/setup.py#L23) but for some reason they were not picked up here :(

additionally, the script needs six, but there is no six in setup.py: https://github.com/latchset/keycloak-httpd-client-install/blob/master/bin/keycloak-httpd-client-install#L16-L17

You will need:
```
Requires:       python-requests
Requires:       python-requests-oauthlib
Requires:       python-jinja2
Requires:       python-six
```

and then probably also `setuptools`: https://github.com/latchset/keycloak-httpd-client-install/blob/master/keycloak_httpd_client/__init__.py#L1 and `lxml`: https://github.com/latchset/keycloak-httpd-client-install/blob/master/keycloak_httpd_client/utils.py#L22